Gopkg.toml

```diff
@@ -43,6 +43,10 @@
   name = "github.com/kubernetes-csi/csi-lib-utils"
   version = "0.1.0"
 
+[[constraint]]
+  name = "k8s.io/klog"
+  revision = "8139d8cb77af419532b33dfa7dd09fbc5f1d344f"
```

Comment:
Same comment about toml versions as the other prs.  @pohly would you like the toml to be cleaned up here or in a separate PR?

Reply:
@msau42 removed klog from toml in latest patch set.